BitwardenShared/Core/Platform/Services/StateService.swift

```diff
@@ -300,12 +300,20 @@ protocol StateService: AnyObject {
     ///
     func getVaultTimeout(userId: String?) async throws -> SessionTimeoutValue
 
+    /// Whether the user is authenticated.
+    ///
+    /// - Parameter userId: The user ID to check if they are authenticated.
+    /// - Returns: Whether the user is authenticated.
+    ///
+    func isAuthenticated(userId: String?) async throws -> Bool
+
     /// Logs the user out of an account.
     ///
-    /// - Parameter userId: The user ID of the account to log out of. Defaults to the active
-    ///   account if `nil`.
+    /// - Parameters:
+    ///   - userId: The user ID of the account to log out of. Defaults to the active account if `nil`.
+    ///   - userInitiated: Whether the logout was user initiated or a result of a logout timeout action.
     ///
-    func logoutAccount(userId: String?) async throws
+    func logoutAccount(userId: String?, userInitiated: Bool) async throws
```

Comment:
was going to suggest having a default value here but it seems we are using both cases about the same.